models/Profile.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+const bcryptjs = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+
+const ProfileSchema = new mongoose.Schema({
+  userName: {
+    type: String,
+    required: [true, "Please provide a name"],
+    minLength: [3, "Name should be more than 3 characters"],
+    maxLength: [50, "Name shouldn't be more than 50 characters"],
+  },

```

Comment:
**Suggestion:** Try using the same name your field has in your error messages. For instance, instead of "Please provide a name", the message could be "Please provide the userName" or "userName is required".
You could also rename your field to match the messages, I think using just `name` in this context would also make sense.